lib/msal-node/src/config/Configuration.ts

```diff
@@ -145,6 +146,7 @@ const DEFAULT_SYSTEM_OPTIONS: Required<NodeSystemOptions> = {
     loggerOptions: DEFAULT_LOGGER_OPTIONS,
     networkClient: new HttpClient(),
     proxyUrl: Constants.EMPTY_STRING,
+    customAgent: new http.Agent() || new https.Agent(),
```

Comment:
What does it mean? Doesn't this just resolve to `new http.Agent()` ?

Reply:
My improvements should address this.